services/api/database/migrations/20241231000100_mysql8_compatibility.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = async function(knex) {
+    return knex.schema
+    .alterTable('advanced_task_definition', function (table) {
+        table.specificType('description', 'text').notNullable().alter();
+        table.string('group_name', 300).alter();
+        table.specificType('command', 'text').alter();
+        table.timestamp('deleted').notNullable().alter();
+    })
+    .alterTable('environment', function (table) {
+        table.timestamp('deleted').notNullable().alter();
+    })
+    .alterTable('environment_fact', function (table) {
```

Comment:
Missing ennvironment_backup deleted field update https://github.com/uselagoon/lagoon/pull/3816/files#diff-94fc09ab7f90affd501ddddf3ae66b4d0b41b4dd22b9b2849f2f692d3b8f3d75R101

Reply:
We dropped the column in a previous migration https://github.com/uselagoon/lagoon/blob/ab7caf5d186d7c530f669732367d45fec4e1107d/services/api/database/migrations/20240507000000_delete_orphaned_backup_resources.js#L16